libs/template/presets/fulfillment/theme-meta/pwa-theme-meta.initializer.ts

```diff
@@ -6,6 +6,9 @@ export class PWAThemeMetaInitializer implements AppInitializer {
 
   initialize(): void {
     this.metaService?.add([
+      {
```

Comment:
wonder if we need this, as it should rely on meta text from the experience data

Reply:
Yep, just re-checked it and we don't need it.